.tekton/pull-request.yaml

```diff
@@ -163,6 +163,82 @@ spec:
         workspaces:
           - name: source
             workspace: workspace
+      - name: get-tasks-to-evaluate
```

Comment:
Might be neater to put this in `.tekton/tasks/ec-check.yaml` similar to how YAML linting is done today.

Reply:
good idea.